recipes/standardese/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "standardese" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tgz
+  sha256: 4f27dbba6d577636a622dd0dc13257e8f11ed7bab5c41808bc0943720c2cc842
+  patches:
+    - 0001-Actually-install-standardese-in-make-install.patch
+    - 0002-boost-link-dynamic.patch
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  run_exports:
+    # standardese uses semantic versioning
+    - {{ pin_subpackage("standardese") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - boost-cpp
+    - clangdev
+  run:
+    - {{ pin_compatible('libclang') }}
+    - {{ pin_compatible('boost-cpp') }}
```

```suggestion
    - boost-cpp
```

Global pinning from conda-forge will apply. (Boost breaks ABI in minor versions and globally boost is pinned to 1.70)

---

Right…I should have known that.